### PR TITLE
Reduce concurrency of QueueGranules

### PR DESCRIPTION
**Pull request by yjpa7145**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -97,6 +97,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Fixed a race condition with bulk granule delete causing deleted granules to still appear in Elasticsearch. Granules removed via bulk delete should now be removed from Elasticsearch.
 - **CUMULUS-1961**
   - Fixed `activeCollections` query only returning 10 results
+- **CUMULUS-xxx**
```
Comment (markdboyd) [inline CHANGELOG.md:100]: I'd just drop this. You finished the work before we even had time to create a ticket

File: CHANGELOG.md [hunk 2]
```diff
@@ -97,6 +97,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+  - Reduced concurrency of `QueueGranules` task, defaulting to `3` and making it
```
Comment (markdboyd) [inline CHANGELOG.md:101]: Can we mention here that it is configurable via `config.concurrency` on the task? I just don't see anywhere else where we advertise available task config?

Comment (yjpa7145) [reply CHANGELOG.md]: fe27da147

File: CHANGELOG.md (hunk 1 above)
Comment (yjpa7145) [reply CHANGELOG.md:100]: Abdelhak is creating a ticket now.

File: CHANGELOG.md (hunk 2 above)
Comment (Jkovarik) [reply CHANGELOG.md]: In theory, we were working on task pages (https://nasa.github.io/cumulus/docs/tasks) (see Tasks sidebar pull down), but there isn't one for this task.     We could add a quick page, but that's out of scope.

**Review by Jkovarik — CHANGES_REQUESTED** [review]
This seems good, however should we consider *using* this config in a workflow and/or add a unit test on the handler to make sure this doesn't get broken in the future?

**Comment by yjpa7145** [comment]
> This seems good, however should we consider _using_ this config in a workflow and/or add a unit test on the handler to make sure this doesn't get broken in the future?

be234b1d0

**Review by Jkovarik — APPROVED** [review]
👍  Looks good

**Pull request merged**